ui/app/styles/core/buttons.scss

```diff
@@ -351,15 +350,14 @@ a.button.disabled {
     color: $ui-gray-500;
   }
 
-  &.icon-only {
-    margin-right: $spacing-xxs;
-    margin-left: $spacing-xxs;
-  }
-
   &.transparent {
     background: none;
-    border: none;
     box-shadow: none;
+    border: 1px solid transparent;
```

Comment:
added so the copy button matches `<Hds::Button>` (the masked input eye hasn't been replaced yet)
![hover-copy](https://github.com/hashicorp/vault/assets/68122737/d82b151d-1c09-46dc-9ba1-3b33f7572dd9)

Reply:
Copy button doesn't have a `@color=tertiary` option so this styling will likely remain